src/runtime/long.cpp

```diff
@@ -385,7 +385,6 @@ Box* longRepr(BoxedLong* v) {
     strcat(buf, "L");
 
     auto rtn = new BoxedString(buf);
-    free(buf);
```

Comment:
I don't think this call to free() should be removed?  We allocate the space with malloc()